app/views/layouts/_analytics.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if Rails.application.config.analytics_tracking_id or Rails.application.config.analytics_test_tracking_id %>
+<% if (Rails.application.config.analytics_tracking_id || Rails.application.config.analytics_test_tracking_id) && !request.url.include?(new_consent_path) %>
```

Comment:
Won't this always return true? (Assuming that analytics_tracking_id is always available)

T || F = T

Reply:
Nope -- it'll retuurn the left half if it is not nil, else the right half -- but yes it will likely be truthy.

HOWEVER && has a higher precedence than || so it can be false.